site/app/controllers/pdf/PDFController.php

```diff
@@ -44,14 +44,13 @@ public function showStudentPDF(string $gradeable_id, string $filename, string $p
                         $pdf_info = explode('_', $no_extension);
                         $pdf_id = implode('_', array_slice($pdf_info, 0, -1));
                         $grader_id = $pdf_info[count($pdf_info) - 1];
-                        if ($pdf_id . '.pdf' === $filename && $grader_id == $grader) {
+                        if ($pdf_id . '.pdf' === $filename) {
```

Comment:
No, but I would argue that it is outside the scope of this PR, since the function was not written in this PR and access control was not removed in this PR.